app/models/course.rb

```diff
@@ -140,8 +140,8 @@ def advance_start_at_duration_days=(value)
   def initialize_duplicate(duplicator, other)
     self.start_at += duplicator.time_shift
     self.end_at += duplicator.time_shift
-    self.title = duplicator.new_course_title
-    self.creator = duplicator.current_user
+    self.title = duplicator.params[:new_course_title]
```

Comment:
Here's an example of documenting options: https://github.com/Coursemology/coursemology2/blob/master/app/helpers/application_cocoon_helper.rb#L14-L22